File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<string> strArr) {
    vector<int> arr(100, 0);
    int answer = 0;
    for(string s : strArr) {
        arr[s.size()] += 1;
    }
    answer = *max_element(arr.begin(), arr.end());
    return answer;
}

File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.java
class Solution {
    public int solution(String[] strArr) {
        int answer = 0;
        int[] list = new int[31];
        for(String a : strArr) {
            list[a.length()]++;
        }
        for(int a : list) {
            if(answer < a) {
                answer = a;
            }
        }
        return answer;
    }
}